capstone/src/test/java/com/google/sps/LessonDeserializationTest.java

```diff
@@ -0,0 +1,219 @@
+package com.google.sps;
+
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+
+import org.junit.Assert;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.sps.data.Form;
+import com.google.sps.data.Lesson;
+import java.io.IOException;
+import com.google.appengine.api.datastore.Entity;
+
+@RunWith(JUnit4.class)
+public final class LessonDeserializationTest {
+
```

Comment:
Created two new files when originally had one LessonTest class to test both serialization and deserialization methods. I believe unit testing means creating a class per method?

Reply:
Got it. Makes a lot more sense